traffic_ops/traffic_ops_golang/api/api.go

```diff
@@ -1075,6 +1102,34 @@ func GetUserFromReq(w http.ResponseWriter, r *http.Request, secret string) (auth
 	return user, nil, nil, http.StatusOK
 }
 
+func getCookieFromAccessToken(bearerToken string, secret string) (*http.Cookie, error) {
+	var cookie *http.Cookie
+	claims := jwt.MapClaims{}
+	token, err := jwt.ParseWithClaims(bearerToken, claims, func(token *jwt.Token) (interface{}, error) {
+		return []byte(secret), nil
+	})
+	if err != nil {
+		return nil, fmt.Errorf("parsing claims: %w", err)
+	}
+	if !token.Valid {
+		return nil, errors.New("invalid token")
+	}
+
+	for key, val := range claims {
+		switch key {
+		case "mojoCookie":
+			if _, ok := val.(string); !ok {
+				return nil, errors.New("invalid token - mojoCookie must be a string")
+			}
+			cookie = &http.Cookie{
+				Value: val.(string),
```

Comment:
Can we just use the value directly from line 1121, instead of type casting it again?
Also, `mojoCookie` could be a constant.